### DISCUSSION TITLE
Can not edit component data

### DISCUSSION THREAD

**Discussion by tmcdos** [vuejs/devtools-v6#2141] [General]
I installed Vue Devtools v6.6.1 and inspecting an application that uses Vue v2. When I try to change the value of any variable in any component - I can type a new value but when I click on the "diskette" icon to the right of the textbox, the value of the component data item does not change.
Why ?

https://github.com/vuejs/devtools/assets/19253115/11121f7b-d683-40e2-8633-e0546cd59ed9

**Comment by thomassth** [comment]
Likely fixed with #2153 , awaiting merge
